plugins/scanners/fossid/src/main/kotlin/FossId.kt

```diff
@@ -920,14 +925,25 @@ class FossId internal constructor(
             snippetChoices,
             snippetLicenseFindings
         )
-        markFilesWithChosenSnippetsAsIdentified(
+        val newlyMarkedFiles = markFilesWithChosenSnippetsAsIdentified(
             scanCode,
             snippetChoices,
             snippetFindings,
             rawResults.listPendingFiles,
             snippetLicenseFindings
         )
 
+        val pendingFilesCount = (rawResults.listPendingFiles - newlyMarkedFiles.toSet()).size
+
+        issues.add(
+            0,
+            Issue(
+                source = name,
+                message = "This scan has $pendingFilesCount file(s) pending identification in FossID.",
+                severity = Severity.HINT
+            )
+        )
```

Comment:
Why not `issues += Issue(...)`?

Reply:
Because I would like to keep the same behavior as before the commit: this "special" issue should be the first of the issues.

Reply:
It would be cool if one could write `issues =+ Issue(...)` to achieve that.

Reply:
I think that does not work as intended, issues are sorted during serialization:

https://github.com/oss-review-toolkit/ort/blob/d7af7364be933aef0b9af20c66c58c7c03867b95/model/src/main/kotlin/ScanSummary.kt#L79-L85
https://github.com/oss-review-toolkit/ort/blob/d7af7364be933aef0b9af20c66c58c7c03867b95/model/src/main/kotlin/ScanSummary.kt#L151-L164

And they can also be sorted again by any reporter. Therefore I would not care about the order at this place.